src/likelihoods.jl

```diff
@@ -0,0 +1,10 @@
+
+"""
+    gaussian(fx::LatentGP, y)
+
+Returns gaussian log-likelihood of output `y` w.r.t to the given `LatentGP`.  
+"""
+function gaussian(fx::LatentGP, y)
+    return Distributions.logpdf(MvNormal(mean(fx), cov(fx) + fx.σ² * I), y)
+end
+
```

Comment:
Under the new design, do we need this?

Reply:
Nope. I have more on this for you on slack.